src/status_im/ui2/screens/communities/communities_home.cljs

```diff
@@ -101,13 +103,26 @@
                                 :description         (i18n/label :t/whats-trending)
                                 :accessibility-label :communities-home-discover-card}])
 
+
+(defn community-card []
```

Comment:
`active-members` instead of `likes-count`
`mutual-contacts` instead of ` dislike-count`